### PR TITLE
community/binutils-avr: add secfixes

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: community/binutils-avr/APKBUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Maintainer: Stefan Wagner <stw@bit-strickerei.de>
 pkgname=binutils-avr
 pkgver=2.32
-pkgrel=0
+pkgrel=1
```
Comment (Ikke) [inline community/binutils-avr/APKBUILD]: Is there a reason for the pkgrel bump here? As far as I know the secfixes don't impact the package, and removing the `-j1` etiher.

Comment (J0WI) [reply community/binutils-avr/APKBUILD]: No, it doesn't really matter here.

**Comment by maxice8** [comment]
Can be closed, merged

**Comment by J0WI** [comment]
2fb8dd928bdd89703d98409e94a1a41fb10c1a34

**Pull request closed without merging**